package.json

```diff
@@ -11,10 +11,13 @@
   },
   "dependencies": {
     "@next/font": "^14.0.2",
+    "cloudinary": "^1.41.0",
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.8",
+    "formidable": "^3.5.1",
     "next": "^13.5.6",
     "react": "18.2.0",
+    "react-awesome-button": "^7.0.5",
```

Comment:
Did you mean to install this for this PR? I don't see any references to it in the code. 

If not you can always uninstall. 

`npm uninstall react-awesome-button`

Reply:
@onemanwenttomow thanks for taking the time to review our code!
To be honest, we don't remember what happened here and that's why we've uninstalled it now..